lib/Parser.js

```diff
@@ -312,16 +312,19 @@ void OnParserTokenResult;
 
 function expected(
   expectedDescription /*: string */,
-  actual /*: any */
+  actual /*: mixed */
```

Comment:
This was the last `any` annotation left after making ElmJson.js (we used to have a bunch of `elmJson: any` parameters to functions). So I got rid of this one as well. That required the `stringify` function to please Flow, since unexpectedly `JSON.stringify(undefined) === undefined` (all other inputs than `undefined` produce a string or throw an error).

Reply:
nice, is there a lint to prevent use of any?

Reply:
There’s [eslint-plugin-flowtype/no-weak-types](https://github.com/gajus/eslint-plugin-flowtype#no-weak-types). I could install that plugin and enable some rules in another PR – I think this one has enough changes already.